packages/manager/apps/web/client/app/hosting/dashboard/hosting.controller.js

```diff
@@ -122,6 +123,7 @@ export default class {
     this.goToDatabaseTab = goToDatabaseTab;
     this.WucUser = WucUser;
     this.HostingDatabase = HostingDatabase;
```

Comment:
HostingDatabase seems to no longer be used in this file, I think you can remove it safely